packages/scandipwa/src/component/ReviewStar/ReviewStar.style.scss

```diff
@@ -9,6 +9,8 @@
     border: 0;
     transition: color 150ms ease-in;
     color: $white;
+    // stylelint-disable declaration-no-important
+    margin-block-start: 0 !important;
```

Comment:
Do not use `!important`, instead use higher priority class to override the rule.